src/FrontendObligationChecker.UnitTests/ViewComponents/CookieBannerViewComponentTests.cs

```diff
@@ -0,0 +1,236 @@
+﻿using FluentAssertions;
+using FrontendObligationChecker.Models.Config;
+using FrontendObligationChecker.Models.Cookies;
+using FrontendObligationChecker.Services.Infrastructure.Interfaces;
+using FrontendObligationChecker.ViewComponents;
+using FrontendObligationChecker.ViewModels;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewComponents;
+using Microsoft.Extensions.Options;
+using Moq;
+
+namespace FrontendObligationChecker.UnitTests.ViewComponents;
+
+[TestClass]
+public class CookieBannerViewComponentTests
+{
+    private const string CookieName = ".epr_cookie_policy";
```

Comment:
Nice to be: CookieName should be all caps for consistency?

Reply:
The consts have different uses - the https://github.com/DEFRA/epr-obligationchecker-frontend/pull/11 is actually more consistent with standards. PATH etc are used because they are placeholders in urls, we can leave as is